data-prepper-core/src/main/java/org/opensearch/dataprepper/core/livecapture/LiveCaptureOutputManager.java

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.core.livecapture;
+
+import org.opensearch.dataprepper.model.event.Event;
+import org.opensearch.dataprepper.model.event.JacksonEvent;
+import org.opensearch.dataprepper.model.record.Record;
+import org.opensearch.dataprepper.model.sink.Sink;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+/**
+ * Manages output of live capture data to a configured sink.
+ * Processes complete live capture traces from individual events and forwards them
+ * to a live capture sink for storage or analysis. This class works with event-bound
+ * live capture data and does not maintain any global state.
+ */
+public class LiveCaptureOutputManager {
+    private static final Logger LOG = LoggerFactory.getLogger(LiveCaptureOutputManager.class);
+    private static volatile LiveCaptureOutputManager INSTANCE;
+    private final AtomicBoolean initialized = new AtomicBoolean(false);
+    private final AtomicBoolean enabled = new AtomicBoolean(false);
+    private volatile Sink<Record<Event>> outputSink;
```

Comment:
I think the outputSink should also be in the LiveCaptureManager

Reply:
Is it not possible to merge LiveCapture Manager and LiveCaptureOutputManager? Why should they be separate?